package.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -euo pipefail
+
+. variables.sh
```

Comment:
nit: expand out `source` for clarity.

```suggestion
source variables.sh
```